common/f3d_materials.md

```diff
@@ -0,0 +1,193 @@
+
+:::::{tab-set}
+
+::::{tab-item} Color Combiner & Sources
+
+Color and alpha values for pixels are determined using the color combiner. The color combiner takes various source value such as textures, solid colors, lights and for each pixel applies a formula of {math}`Color = (A-B)*C + D` to get a resultant value out for both color and alpha. This value is sent to the blender for combining the pixel and alpha with existing pixel colors or additional inputs such as fog.
+
+## Color Combiner
+In the F3D material inspector, the Combiner tab offers you a selection of 8 inputs in one cycle mode, and 16 in two cycle mode; by default one cycle mode is selected. The inputs in the left represent the RGB color that will be used for blending color, while the ones on the left aptly labeled "Alpha" will be for alpha blending.
+
+Each color channel independently performs math operations following the color combiner formula. When given an RGB input, the color will use each channel as its own value, but when given a constant value, such as 0, 1 or a grayscale texture, then all channels will be given that value as input.
+
+When in two cycle mode, two sets of combiner inputs appear. Values from the output of the first cycle are output to the second cycle under the variable called [combined]{.key_text}.
+
+:::{admonition} Note
+The value of `combined` is not valid in 1 cycle, and using it will result in noise patterns and unmapped values being used.
+:::
+
+:::{admonition} Note
+When in 2 cycle mode, `Texel0` and `Texel1` are swapped, and `Texel1` maps to the next pixel in memory.
+:::
+
+## Sources
+
+Each input available to edit will be one of the inputs selected in the combiner tab. Inputs available can be one of following types:
+
+## Textures
+
+Textures map the output of the source image to pixels using UVs. There are several texture types available to use, and many properties which is covered in their own article: (link I guess).
+
+The combiner supports two textures for use, named [Texture 0]{.key_text} and [Texture 1]{.key_text}. You can load up to 4kb into texture memory (TMEM) at once using any type but Color Indexed (CI), which only supports 2kb, the other 2kb is reserved for palettes.
+
+Fast64 will calculate TMEM usage when selecting a texture. If you're texture is too large, you will have to resize it, or use **Large Texture Mode**. When using large textures, you will have to subdivide the mesh so that there are an equal number of faces as TMEM size divisions; this can be done automatically using the **Create Large Texture Mesh** tool which is located in the Fast64 Tools panel in the 3D viewport.
```

Comment:
```suggestion
Fast64 will calculate TMEM usage when selecting a texture. If your texture is too large, you will have to resize it, or use **Large Texture Mode**. When using large textures, you will have to subdivide the mesh so that there are an equal number of faces as TMEM size divisions; this can be done automatically using the **Create Large Texture Mesh** tool which is located in the Fast64 Tools panel in the 3D viewport.
```